benches/tree_hash_root.rs

```diff
@@ -43,6 +43,26 @@ pub fn tree_hash_root(c: &mut Criterion) {
             })
         },
     );
+
+    c.bench_with_input(
+        BenchmarkId::new("tree_hash_root_list_parallel", size),
+        &size,
+        |b, &size| {
+            b.iter(|| {
+                let l1 = List::<u64, C>::try_from_iter(0..size).unwrap();
+                let mut l2 = List::<u64, C>::try_from_iter(0..size).unwrap();
```

Comment:
let's try initialising l2 by cloning l1, then they will share a bunch of nodes in common and the parallel hashing will be forced to either duplicate work (as it does now) or block (as it would with the upgradeable lock)

Reply:
Ooh yeah good point